lib/core/info-drawer/info-drawer.component.stories.ts

```diff
@@ -0,0 +1,276 @@
+            control: 'text',
+            description: 'Text content of the second tab (Tab Layout only)',
+            defaultValue: undefined,
+            table: {
+                type: { summary: 'string' },
+                defaultValue: { summary: 'undefined' }
+            }
+        },
+        tab3Text: {
+            control: 'text',
+            description: 'Text content of the third tab (Tab Layout only)',
+            defaultValue: undefined,
+            table: {
+                type: { summary: 'string' },
+                defaultValue: { summary: 'undefined' }
+            }
+        },
+        cardText: {
+            control: 'text',
+            description: 'The content of the single card (Single Layout only)',
+            defaultValue: undefined,
+            table: {
+                type: { summary: 'string' },
+                defaultValue: { summary: 'undefined' }
+            }
+        }
+    }
+} as Meta;
+
+const tabLayoutTemplate: Story<InfoDrawerModule> = (args: InfoDrawerComponent) => ({
+    props: {
+        ...args,
+        currentTab: action('currentTab')
+    },
+    template:
+        `<adf-info-drawer title="Activities" [showHeader]="showHeader" (currentTab)="currentTab($event)">
+            <div info-drawer-buttons>
+                <mat-icon>clear</mat-icon>
+            </div>
+
+            <adf-info-drawer-tab [label]="label1" [icon]="[icon1]">
+                <div class="info-drawer-tab-text">{{ tab1Text }}</div>
+            </adf-info-drawer-tab>
+
+            <adf-info-drawer-tab [label]="label2" [icon]="[icon2]" *ngIf="showSecondTab">
+                <div class="info-drawer-tab-text">{{ tab2Text }}</div>
+            </adf-info-drawer-tab>
+
+            <adf-info-drawer-tab [label]="label3" [icon]="[icon3]" *ngIf="showThirdTab">
+                <div class="info-drawer-tab-text">{{ tab3Text }}</div>
+            </adf-info-drawer-tab>
+        </adf-info-drawer>`
+});
+
+const singleLayoutTemplate: Story<InfoDrawerModule> = (args: InfoDrawerComponent) => ({
+    props: args,
+    template:
+        `<adf-info-drawer title="Single Activities" [showHeader]="showHeader">
+            <div info-drawer-title>File info</div>
+
+            <div info-drawer-buttons>
+                <mat-icon>clear</mat-icon>
+            </div>
+
+            <div info-drawer-content>
+                <mat-card>
+                    {{ cardText }}
+                </mat-card>
+            </div>
+        </adf-info-drawer>`
+});
+
+export const tabLayoutWithTextLabels = tabLayoutTemplate.bind({});
+tabLayoutWithTextLabels.args = {
+    title: 'Activities',
+    showSecondTab: true,
+    showThirdTab: true,
+    label1: 'Activity',
+    label2: 'Details',
+    label3: 'More Info',
+    tab1Text: `This is a variant of the Info Drawer Layout component that displays information in tabs. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam urna odio, sagittis vel nulla vel, condimentum egestas dolor.
```

Comment:
can we move those long texts to file with mocks to avoid duplications? i suggest to create mock folder inside info-drawer, and inside it create a mock file.

Reply:
Sure, no problem 🤠